pages/globalHackaton/gatsby-node.js

```diff
@@ -18,19 +18,26 @@ exports.createPages = async ({ graphql, actions }) => {
           node {
             id
             uid
+            data {
+              link_to_event {
+                url
+              }
+            }
           }
         }
       }
     }
   `);
   const template = path.resolve('./src/templates/location.js');
   pages.data.allPrismicLocations.edges.forEach(edge => {
```

Comment:
We should really start to use TypeScript or Flow, because almost everything which is coming from GraphQL is nullable.

Reply:
Agree